Adding user invite status release note (#8615)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

- **New**: The [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now supports [Omni](https://docs.omni.co/integrations/dbt/semantic-layer) as a partner integration. For more info, see [Available integrations](/docs/cloud-integrations/avail-sl-integrations).
- **Enhancement**: We clarified documentation for cumulative log size limits on run endpoints, originally introduced in [October 2025](/docs/dbt-versions/2025-release-notes#october-2025). When logs exceed the cumulative size limit, dbt omits them and displays a banner. No functional changes were made in February 2026. For more information, see [Run visibility](/docs/deploy/run-visibility#log-size-limits).
- **New**: The `immutable_where` configuration is now supported for Snowflake dynamic tables. For more information, see [Snowflake configurations](/reference/resource-configs/snowflake-configs#immutable-where).

> - **Fix**: The user invite details now shows more details in invite status, giving admins visibility into users who accepted an invite to an SSO-protected account but haven't yet logged in via SSO. Previously, these invites were hidden, making it appear as if the user hadn't been invited. 

Reviewer: Reducing repetition
```suggestion
- **Fix**: The user invite details now show more information in invite status, giving admins visibility into users who accepted an invite to an SSO-protected account but haven't yet logged in via SSO. Previously, these invites were hidden, making it appear as if the user hadn't been invited. 
```

The Invites endpoints of the dbt platform Admin v2 API now include these additional statuses:
  - `4` (PENDINGEMAIL_VERIFICATION)
  - `5` (EMAIL_VERIFIED_SSO).

> - **Enhancement**: Improved performance on run details in dbt platform when connecting with GCP.

Author: ```suggestion
- **Enhancement**: Improved performance on Runs endpoint for dbt platform Admin V2 and run details in dbt platform when connecting with GCP.
```

> - **Enhancement**: Improved performance on Runs endpoint for dbt platform Admin V2 and run details in dbt platform when connecting with GCP.

Author: ```suggestion
- **Enhancement**: Improved performance on Runs endpoint for dbt platform Admin V2 API and run details in dbt platform when connecting with GCP.
```

> - **Enhancement**: Improved performance on Runs endpoint for dbt platform Admin V2 API and run details in dbt platform when connecting with GCP.

Reviewer: Removing some repetition. 
```suggestion
- **Enhancement**: Improved performance on Runs endpoint for Admin V2 API and run details in dbt platform when connecting with GCP.
```

- **New**: The [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now supports [Omni](https://docs.omni.co/integrations/dbt/semantic-layer) as a partner integration. For more info, see [Available integrations](/docs/cloud-integrations/avail-sl-integrations).
- **Enhancement**: We clarified documentation for cumulative log size limits on run endpoints, originally introduced in [October 2025](/docs/dbt-versions/2025-release-notes#october-2025). When logs exceed the cumulative size limit, dbt omits them and displays a banner. No functional changes were made in February 2026. For more information, see [Run visibility](/docs/deploy/run-visibility#log-size-limits).
- **New**: The `immutable_where` configuration is now supported for Snowflake dynamic tables. For more information, see [Snowflake configurations](/reference/resource-configs/snowflake-configs#immutable-where).

> - **Fix**: The user invite details now shows more details in invite status, giving admins visibility into users who accepted an invite to an SSO-protected account but haven't yet logged in via SSO. Previously, these invites were hidden, making it appear as if the user hadn't been invited.

Author: ```suggestion
- **Fix**: The user invite details now shows more details in invite status, giving admins visibility into users who accepted an invite to an SSO-protected account but haven't yet logged in via SSO. Previously, these invites were hidden, making it appear as if the user hadn't been invited. The Invites endpoints of the dbt platform Admin v2 API now include these additional statuses: 4 -PENDING_EMAIL_VERIFICATION
5 - EMAIL_VERIFIED_SSO.
```

- **New**: The [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now supports [Omni](https://docs.omni.co/integrations/dbt/semantic-layer) as a partner integration. For more info, see [Available integrations](/docs/cloud-integrations/avail-sl-integrations).
- **Enhancement**: We clarified documentation for cumulative log size limits on run endpoints, originally introduced in [October 2025](/docs/dbt-versions/2025-release-notes#october-2025). When logs exceed the cumulative size limit, dbt omits them and displays a banner. No functional changes were made in February 2026. For more information, see [Run visibility](/docs/deploy/run-visibility#log-size-limits).
- **New**: The `immutable_where` configuration is now supported for Snowflake dynamic tables. For more information, see [Snowflake configurations](/reference/resource-configs/snowflake-configs#immutable-where).

> - **Fix**: The user invite details now shows more details in invite status, giving admins visibility into users who accepted an invite to an SSO-protected account but haven't yet logged in via SSO. Previously, these invites were hidden, making it appear as if the user hadn't been invited. The Invites endpoints of the dbt platform Admin v2 API now include these additional statuses: 4 -PENDING_EMAIL_VERIFICATION

Reviewer: ```suggestion
- **Fix**: The user invite details now shows more details in invite status, giving admins visibility into users who accepted an invite to an SSO-protected account but haven't yet logged in via SSO. Previously, these invites were hidden, making it appear as if the user hadn't been invited. The Invites endpoints of the dbt platform Admin v2 API now include these additional statuses: 
- 4 - PENDING_EMAIL_VERIFICATION
```

> 5 - EMAIL_VERIFIED_SSO.

Reviewer: ```suggestion
- 5 - EMAIL_VERIFIED_SSO.
```
